packages/kbn-zod/README.md

```diff
@@ -0,0 +1,4 @@
+# `@kbn/zod`
+
+Kibana's `Zod` library. Exposes the `Zod` API with some Kibana-specific
+improvements.
```

Comment:
Is it possible to document what these improvements are?

Reply:
We don't have any improvement for now. My initial thinking was mostly about OpenAPI improvements, maybe something like [zod-openapi](https://github.com/samchungy/zod-openapi) (I still don't know if using this library is a good idea or maybe it is better to get inspiration from it and implement it ourselves and in the meantime try to contribute to Zod as well).

So, basically, this is a placeholder to which we can add improvements as we progress.